docs/learn-teleport.md

```diff
@@ -0,0 +1,41 @@
+---
+id: learn-teleport
+title: Teleporting Assets To Other Parachains
+sidebar_label: Teleporting Assets
+---
+
+One of the main properties Polkadot and Kusama brings to ecosystem is blockchain interoperability.
```

Comment:
Perhaps for clarity sake:
`One of the main properties of the Polkadot and Kusama ecosystem is blockchain interoperability.`
would even add:
`One of the main properties of the Polkadot and Kusama ecosystem is _true_ blockchain interoperability.`
... 
`One of the main properties of the Polkadot and Kusama protocol is the enablement of true blockchain interoperability.`